### ISSUE TITLE
monit -v mode overpowers with useless info

### ISSUE THREAD

**Issue by dbabits** [dbabits/monit#3]
example, even with an all but empty configuration (nothing being monitored), monit -v dumps this stuff every second, filling the log and obstructing the useful info.
Without -v it's silent and doesn't output anything.
This is the stuff I need, and it's only available with -v:
```
'LongRunningCheck' status check deferred - waiting on program to exit
'LongRunningCheck' status succeeded (0) -- no output
'LongRunningCheck' program started
'LongRunningCheck' status check deferred - waiting on program to exit
```
Below is the stuff I want to get rid of:
1. Why is it looking for all procs running on the system, when not asked to do so?

1. Does it expect to be running as root? (Docs don't say it)

1. Is there a way to turn it off?

```
Cannot open proc file '/proc/9011/io' -- Permission denied
Cannot read proc file '/proc/9011/attr/current' -- Invalid argument
Cannot open proc file '/proc/9012/io' -- Permission denied
Cannot read proc file '/proc/9012/attr/current' -- Invalid argument
Cannot open proc file '/proc/9013/io' -- Permission denied
Cannot read proc file '/proc/9013/attr/current' -- Invalid argument
Cannot open proc file '/proc/9017/io' -- Permission denied
Cannot read proc file '/proc/9017/attr/current' -- Invalid argument
Cannot open proc file '/proc/902/io' -- Permission denied
Cannot read proc file '/proc/902/attr/current' -- Invalid argument
Cannot open proc file '/proc/909/io' -- Permission denied
Cannot read proc file '/proc/909/attr/current' -- Invalid argument
Cannot open proc file '/proc/912/io' -- Permission denied
Cannot read proc file '/proc/912/attr/current' -- Invalid argument
Cannot open proc file '/proc/915/io' -- Permission denied
Cannot read proc file '/proc/915/attr/current' -- Invalid argument
Cannot read proc file '/proc/9348/attr/current' -- Invalid argument
Cannot open proc file '/proc/940/io' -- Permission denied
Cannot read proc file '/proc/940/attr/current' -- Invalid argument
Cannot open proc file '/proc/941/io' -- Permission denied
Cannot read proc file '/proc/941/attr/current' -- Invalid argument
Cannot open proc file '/proc/943/io' -- Permission denied
Cannot read proc file '/proc/943/attr/current' -- Invalid argument
Cannot open proc file '/proc/946/io' -- Permission denied
Cannot read proc file '/proc/946/attr/current' -- Invalid argument
Cannot open proc file '/proc/952/io' -- Permission denied
Cannot read proc file '/proc/952/attr/current' -- Invalid argument
Cannot open proc file '/proc/958/io' -- Permission denied
Cannot read proc file '/proc/958/attr/current' -- Invalid argument
Cannot read proc file '/proc/9647/attr/current' -- Invalid argument
Cannot open proc file '/proc/9669/io' -- Permission denied
Cannot read proc file '/proc/9669/attr/current' -- Invalid argument
Cannot open proc file '/proc/9700/io' -- Permission denied
Cannot read proc file '/proc/9700/attr/current' -- Invalid argument
Cannot open proc file '/proc/9702/io' -- Permission denied
Cannot read proc file '/proc/9702/attr/current' -- Invalid argument
Cannot open proc file '/proc/9730/io' -- Permission denied
Cannot read proc file '/proc/9730/attr/current' -- Invalid argument
Cannot open proc file '/proc/9759/io' -- Permission denied
Cannot read proc file '/proc/9759/attr/current' -- Invalid argument
```

**Comment by ketan-cardup** [comment]
@dbabits  Were you able to solve this issue ? I am facing the same one.

**Comment by dbabits** [comment]
no, I stopped using monit
